src/foam/nanos/crunch/ui/CapableObjectWizardlet.js

```diff
@@ -59,8 +61,8 @@ foam.CLASS({
   methods: [
     {
       name: 'save',
-      code: function() {
-        //
+      code: async function() {
```

Comment:
Can this method be removed?

Reply:
It needs to return a promise, but I'll move it to `BaseWizardlet` since that makes more sense.